app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
Not really.  It's fundamentally bad form and I cannot support it.  If you want, I would remove ALL other references to overriding private methods.  I would *consider* someone else's PR to make all such methods `public`, but only with significant skepticism given how scattershot they seem to be.  

We are trying to be **better** than sufia.  Keeping people dependent on hacks is not doing them any favors.